src/WebJobs.Script/Diagnostics/Extensions/LoggerExtension.cs

```diff
@@ -86,9 +86,9 @@ internal static class LoggerExtension
                 new EventId(314, nameof(FunctionMetadataManagerLoadingFunctionsMetadata)),
                 "Loading functions metadata");
 
-        private static readonly Action<ILogger, int, Exception> _functionMetadataManagerFunctionsLoaded =
+        private static readonly Action<ILogger, int, Exception> _functionMetadataProviderFunctionsLoaded =
```

Comment:
I think the changes to this field/method were unintentional? It's only used by the FunctionMetadataManager class in one place, and you're not passing in a provider or anything as in the other log messages. I think you should revert the changes to this one.